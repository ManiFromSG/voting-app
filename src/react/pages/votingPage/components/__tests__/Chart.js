import Chart from '../chart';
import renderer from 'react-test-renderer';
import React from 'react';

const question = 'Which is your favorite JS library?';
const chartData = [
    ['React', 10],
    ['Angular', 8],
    ['jQuery', 6]
];
const totalVotes = 24;

describe(`The Chart Component`, ()=>{
   it(`Should render successfully`, ()=> {
        const tree = renderer.create(
            <Chart question={question} chartData={chartData} totalVotes={totalVotes} />
        );

        expect(tree.toJSON()).toMatchSnapshot();
   });
});
