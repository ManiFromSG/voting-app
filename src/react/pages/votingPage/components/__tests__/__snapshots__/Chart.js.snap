// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Chart Component Should render successfully 1`] = `
<div
  className="chartComponent"
>
  <h4>
    Which is your favorite JS library?
  </h4>
  <div>
    Total votes: 
    24
  </div>
</div>
`;
